docs/assets/dev/js/editor/document/dynamic/readme.md

```diff
@@ -0,0 +1,119 @@
+## Component -- `$e.components.get('document/dynamic')`
+
+*  **Name**: Dynamic.
+*  **Description**: Provide a way to _enable_, _change_, _disable_ dynamic settings, associated with [Dynamic Tags](https://developers.elementor.com/dynamic-tags/),
+
+## Component `document/dynamic/` -- Commands
+| Command                                                         | Access                                  | Description
+|-----------------------------------------------------------------|-----------------------------------------|-----------------------------------------
+| [Disable](#disable-command----erundocumentdynamicdisable)       | `$e.run('document/dynamic/disable')`    | Disable dynamic tag.
+| [Enable](#enable-command----erundocumentdynamicenable)          | `$e.run('document/dynamic/enable')`     | Enable dynamic tag.
+| [Settings](#settings-command----erundocumentdynamicsettings)    | `$e.run('document/dynamic/settings')`   | Edit dynamic settings.
+
+
+## Disable _Command_ -- `$e.run('document/dynamic/disable')`
+*  **Name**: Disable.
+*  **Description**: Disable dynamic settings.
+*  **Returns**: `{void}`
+*  **Arguments**:
+
+   | Property     | Type                  | Requirement   | Description |
+   |---           |---                    |---            |---|
+   | _container_  | `{Container}`         | **required**  | Container to disable.
+   | _containers_ | `{Array.<Container>}` | **required**  | Containers to disable.
```

Comment:
Also in other places.

```suggestion
   | _container_  | `{Container}`         | **required** (if `containers` isn't supplied)  | Container to disable.
   | _containers_ | `{Array.<Container>}` | **required** (if `container` isn't supplied)  | Containers to disable.
```

Reply:
IDK what's the convention in other places, but maybe change it like that?

```suggestion
   | _container_  | `{Container}`         | **required**  | Container to disable.
   | _containers_ | `{Container[]}` | **required**  | Containers to disable.
```

Reply:
Seems like we already added convention like you suggest so we have to follow it now, thanks 👍🏻